docs/units/decomposing_and_composing.rst

```diff
@@ -123,22 +123,25 @@ Examples
 To convert between unit systems::
 
    >>> u.Pa.to_system(u.cgs)
-   [Unit("10 P / s"), Unit("10 Ba")]
+   [Unit("10 Ba"), Unit("10 P / s")]
 
 There is also a shorthand for this which only returns the first of
 many possible matches::
 
    >>> u.Pa.cgs
-   Unit("10 P / s")
+   Unit("10 Ba")
 
 This is equivalent to decomposing into the new system and then
 composing into the most complex units possible, though
 :meth:`~astropy.units.core.UnitBase.to_system` adds some extra logic to
-return the results sorted in the most useful order::
+return the results sorted with a preference for base units::
```

Comment:
But it's not preferring base units. `to_system()` prefers the unit for the appropriate physical type, if one exists.

Reply:
Rephrased